controller/libpostal.js

```diff
@@ -0,0 +1,30 @@
+const text_analyzer = require('pelias-text-analyzer');
+const _ = require('lodash');
+const iso3166 = require('iso3166-1');
+
+function setup(should_execute) {
+  function controller( req, res, next ){
+    // bail early if req/res don't pass conditions for execution
+    if (!should_execute(req, res)) {
+      return next();
+    }
```

Comment:
I already have plans to revisit express to take advantage of routes and other things instead of these predicates which are getting heavy.  Hiding all that in route jumping will also be heavy, however.  I'll take care of this in a follow-up PR.